setup.py

```diff
@@ -861,6 +861,33 @@ def run(self) -> None:
                 target_hipblaslt_dir = target_dir / "hipblaslt"
                 target_hipblaslt_dir.mkdir(parents=True, exist_ok=True)
                 self.copy_tree(hipblaslt_dir, str(target_hipblaslt_dir))
+
+                # Copy libomp140.x86_64.dll to torch/lib since rocalution.dll needs it
+                omp_name = 'libomp140.x86_64.dll'
+                program_files = os.environ["ProgramFiles(x86)"]
+                vswhere_path = program_files + r'\Microsoft Visual Studio\Installer\vswhere.exe'
+                # Getting libomp140.x86_64.dll installation path through `vswhere`
+                cmd = [
+                    vswhere_path,
+                    "-prerelease",
+                    "-latest",
+                    "-find", rf'VC\Redist\MSVC\**\{omp_name}'
+                ]
+                try:
+                    result = subprocess.check_output(cmd, stderr=subprocess.STDOUT, shell=True, text=True)
+                    dll_paths = result.strip().split('\n')
+                    if not dll_paths:
+                        raise FileNotFoundError("No OpenMP DLL found")
+                    # Get first valid path (latest version)
+                    omp_path = next(p for p in dll_paths if p.strip())
+                    if not os.path.exists(omp_path):
+                        raise FileNotFoundError(f"DLL not found at {omp_path}")
```

Comment:
We aren't yet including rocalution in https://github.com/ROCm/TheRock, and I'm not sure if we would want to bundle libomp even if we were. libomp is in an unstable spot right now, Without build support on Windows in https://github.com/ROCm/llvm-project (see also https://github.com/ROCm/TheRock/issues/711#issuecomment-2914185744). If it's needed, I think we need to build it and actually support it, not just pull it from Visual Studio.